### DISCUSSION TITLE
Dodanie statusu "z nieodczytanymi wiadomościami" do dyskusji.

### DISCUSSION THREAD

**Discussion by MazurKamil** [allegro/allegro-api#4581] [Suggestions - resolved]
:bulb: **Sugestia**

Aktualnie możliwe jest tylko otrzymywanie zwrotki ze statusami [ "CLOSED" "ONGOING" "UNRESOLVED" ] 

![image](https://user-images.githubusercontent.com/49535947/120480646-91ef3c00-c3af-11eb-8f04-719c72a3c401.png)

Po stronie Allegro za to widnieje taki rodzaj poziomowania zgłoszenia: 
![image](https://user-images.githubusercontent.com/49535947/120480909-d975c800-c3af-11eb-8203-752823d5b67a.png)

W takim wypadku czy jest możliwe wprowadzenie takiego statusu też tutuaj?

**Comment by PrzemyslawLukanowski** [comment]
Dyskusje przenosimy do Centrum wiadomości jako [Problemy z zakupem](https://developer.allegro.pl/news/centrum-wiadomosci-rozszerzamy-zasoby-messaging-o-obsluge-problemow-z-zakupem-8dgb9EElMIv). W wersji beta zasobu [GET /messaging/threads](https://developer.allegro.pl/documentation#tag/Message-Center/operation/listThreadsGET) znajdziecie parametr **read**, dzięki któremu możecie filtrować wątki według tego, czy zostały odczytane.

W związku z tym aktualizuję status sugestii jako "rozwiązana".
